ethereumj-core/build.gradle

```diff
@@ -139,7 +139,7 @@ dependencies {
     compile "com.h2database:h2:1.4.187"
     compile "com.typesafe:config:1.2.1"
 
-    compile "org.mapdb:mapdb:2.0-beta12"
+    compile "org.mapdb:mapdb:2.0-beta13"
 
     compile "org.slf4j:slf4j-log4j12:${slf4jVersion}"
     compile "log4j:apache-log4j-extras:${log4jVersion}"
```

Comment:
We decided that we would better avoid unreasonable lib versions upgrades due to possible regressions unless there are desired bug fixes or well known security fixes